Octokit.Tests.Integration/Clients/RepositoryInvitationsClientTests.cs

```diff
@@ -70,6 +179,158 @@ public class TheGetAllForCurrentMethod
+                    Assert.Equal(InvitationPermissionType.Write, response.Permissions);
+                }
+
+                var startOptions = new ApiOptions
+                {
+                    PageSize = 5,
+                    PageCount = 1,
+                    StartPage = 2
+                };
+
+
+                var invitations = await github.Repository.Invitation.GetAllForCurrent(startOptions);
+                Assert.Equal(1, invitations.Count);
+            }
+            finally
+            {
+                if (contexts != null)
+                {
+                    foreach (var context in contexts)
+                    {
+                        context?.Dispose();
+                    }
+                }
+            }
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfInvitationsWithoutStart()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoNames = Enumerable.Range(0, 6).Select(i => Helper.MakeNameWithTimestamp($"public-repo{i}")).ToList();
+
+            RepositoryContext[] contexts = null;
+            try
+            {
+                contexts = repoNames.Select( x =>  github.CreateRepositoryContext(new NewRepository(x)).Result).ToArray();
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest(Permission.Push);
+
+                // invite a collaborator to all repos
+                foreach (var context in contexts)
+                {
+                    var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, context.RepositoryOwner, permission);
+
+                    Assert.Equal(context.RepositoryOwner, response.Invitee.Login);
+                    Assert.Equal(InvitationPermissionType.Write, response.Permissions);
+                }
+
+                var startOptions = new ApiOptions
+                {
+                    PageSize = 5,
+                    PageCount = 1
+                };
+
+
+                var invitations = await github.Repository.Invitation.GetAllForCurrent(startOptions);
+                Assert.Equal(5, invitations.Count);
+            }
+            finally
+            {
+                if (contexts != null)
+                {
+                    foreach (var context in contexts)
+                    {
+                        context?.Dispose();
+                    }
+                }
+            }
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsDistinctInvitationsBasedOnStart()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoNames = Enumerable.Range(0, 10).Select(i => Helper.MakeNameWithTimestamp($"public-repo{i}")).ToList();
+
+            RepositoryContext[] contexts = null;
+            try
+            {
+                contexts = repoNames.Select(x => github.CreateRepositoryContext(new NewRepository(x)).Result).ToArray();
```

Comment:
can you please fix this `.Result` like you did for the other one?

Reply:
Done.